defold-spine/src/script_spine.cpp

```diff
@@ -780,6 +821,7 @@ namespace dmSpine
             {"set_ik_target_position",  SpineComp_SetIKTargetPosition},
             {"set_ik_target",           SpineComp_SetIKTarget},
             {"reset_ik_target",         SpineComp_ResetIK},
+            {"set_bone_position",       SpineComp_SetBonePosition},
```

Comment:
In our api, we use "world" in the names (See [go.get_world_position()](https://defold.com/ref/stable/go/#go.get_world_position:[id]))
This function should be called `set_world_bone_position()`.

That, or make it accept local space coordinate.